starboard/shared/posix/socket_receive_from.cc

```diff
@@ -50,7 +51,7 @@ int SbSocketReceiveFrom(SbSocket socket,
                                &sock_addr.length);
       if (result < 0) {
         SB_LOG(ERROR) << __FUNCTION__
-                      << ": getpeername failed, errno = " << errno;
+                      << ": getpeername failed, errno = " << strerror(errno);
```

Comment:
Consider if you need the `errno` -> `strerror(errno)` changes here. I know it makes logs more readable, but also who knows why `strerror` may end up being crashy on Android + Starboard ?

Reply:
Hmm... That is a fair point because this is `SB_LOG()` which doesn't appear to become a no-op in gold builds.

I'm leaning towards putting this inside an `#if` block that removes them completely on gold builds.